### PR TITLE
Redesign the runtime database. 

### PR DESCRIPTION
**Pull request by runzexia**

### CODE REVIEW

File: pkg/db/schema/runtime/V0_1__init.sql [hunk 1]
```diff
@@ -1,34 +1,55 @@
...
+CREATE INDEX runtime_label_label_key_idx
+		ON runtime_label (label_key);
+CREATE INDEX runtime_label_label_value_idx
+		ON runtime_label (label_value);
+CREATE INDEX runtime_label_create_time_idx
+		ON runtime_label (create_time);
+
+CREATE TABLE runtime_credential (
+	runtime_credential_id VARCHAR(50) PRIMARY KEY             NOT NULL,
+	content               JSON,
```
Comment (zheng1) [inline pkg/db/schema/runtime/V0_1__init.sql]: `content ` is `NOT NULL`

File: pkg/db/schema/runtime/V0_1__init.sql [hunk 2]
```diff
@@ -1,34 +1,55 @@
...
+
+CREATE INDEX runtime_zone_idx
+		ON runtime (zone);
+
+CREATE INDEX runtime_owner_idx
+		ON runtime (owner);
+
+CREATE INDEX runtime_status_idx
+		ON runtime (status);
+
```
Comment (zheng1) [inline pkg/db/schema/runtime/V0_1__init.sql]: `create_time ` and `status_time ` should add index for sorting.

File: pkg/models/runtime_env.go [hunk 3]
```diff
@@ -12,7 +12,7 @@ import (
 	"openpitrix.io/openpitrix/pkg/utils"
 )
 
-const RuntimeEnvTableName = "runtime_env"
+const RuntimeEnvTableName = "runtime"
```
Comment (chilianyi) [inline pkg/models/runtime_env.go]: Remove Env in next pr.

**Comment by runzexia** [comment]
Done

**Pull request merged**